content/id/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -0,0 +1,234 @@
+
+- menjalankan daemon penyimpanan di klaster, seperti `glusterd`, `ceph`, di
+  setiap node.
+- menjalankan daemon pengumpulan log di semua node, seperti `fluentd` atau
+  `logstash`.
+- menjalankan daemon pemantauan node di setiap node, seperti [Prometheus Node Exporter](https://github.com/prometheus/node_exporter), [Flowmill](https://github.com/Flowmill/flowmill-k8s/), [Sysdig Agent](https://docs.sysdig.com), `collectd`, [Dynatrace OneAgent](https://www.dynatrace.com/technologies/kubernetes-monitoring/), [AppDynamics Agent](https://docs.appdynamics.com/display/CLOUD/Container+Visibility+with+Kubernetes), [Datadog agent](https://docs.datadoghq.com/agent/kubernetes/daemonset_setup/), [New Relic agent](https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration), Ganglia `gmond` atau [Instana Agent](https://www.instana.com/supported-integrations/kubernetes-monitoring/).
+
+Dalam kasus sederhana, satu DaemonSet, mencakup semua node, akan digunakan untuk
+setiap jenis daemon. Pengaturan yang lebih rumit bisa saja menggunakan lebih
+dari satu DaemonSet untuk satu jenis daemon, tapi dengan _flag_ dan/atau
+`requests` cpu/memori yang berbeda untuk jenis `hardware` yang berbeda.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Menulis Spec DaemonSet
+
+### Buat DaemonSet
+
+Anda bisa definisikan DaemonSet dalam berkas YAML. Contohnya, berkas
+`daemonset.yaml` di bawah mendefinisikan DaemonSet yang menjalankan _image_ Docker
+fluentd-elasticsearch:
+
+{{< codenew file="controllers/daemonset.yaml" >}}
+
+* Buat DaemonSet berdasarkan berkas YAML:
+```
+kubectl apply -f https://k8s.io/examples/controllers/daemonset.yaml
+```
+
+### _Field_ Wajib
+
+Seperti semua konfigurasi Kubernetes lainnya, DaemonSet membutuhkan _field_
+`apiVersion`, `kind`, dan `metadata`.  Untuk informasi umum tentang berkas konfigurasi, lihat dokumen [deploying applications](/docs/user-guide/deploying-applications/),
+[pengaturan kontainer](/docs/tasks/), dan [pengelolaan objek dengan kubectl](/docs/concepts/overview/working-with-objects/object-management/).
+
+DaemonSet juga membutuhkan bagian [`.spec`](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
+
+### Templat Pod
+
+`.spec.template` adalah salah satu _field_ wajib di dalam `.spec`.
+
+`.spec.template` adalah sebuah [templat pod](/docs/concepts/workloads/pods/pod-overview/#templat-pod). Skemanya benar-benar sama dengan [Pod](/docs/concepts/workloads/pods/pod/), kecuali bagian bahwa ia bersarang/_nested_ dan tidak memiliki `apiVersion` atau `kind`.
+
+Selain _field_ wajib untuk Pod, templat pod di DaemonSet harus
+menspesifikasikan label yang sesuai (lihat [selektor pod](#selektor-pod)).
+
+Templat pod di DaemonSet harus memiliki [`RestartPolicy`](/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy)
+yang bernilai `Always`, atau tidak dispesifikasikan, sehingga _default_ menjadi `Always`.
```

Comment:
What does “Always” mean in bahasa Indonesia? (I recommend explaining the translation).

Reply:
probably helpful if we add extra sentence:
DaemonSet dengan nilai `Always` membuat Pod akan selalu di-_restart_ saat kontainer keluar atau terjadi _crash_.

Reply:
Okie dokie, will add.